components/plan-build-ps1/bin/hab-plan-build.ps1

```diff
@@ -2155,7 +2155,7 @@ try {
 
     # Test to ensure package name and origin contain only valid characters
     foreach ($var in @("pkg_name", "pkg_origin")) {
-      if (-Not ((Get-Content Variable:\$var) -match '^[A-Za-z0-9_-]+$')) {
+      if (-Not ((Get-Content Variable:\$var) -match '^[A-Za-z0-9_.-]+$')) {
```

Comment:
I think you need to backslash escape the `.` here, right?  Otherwise `.` stands for any character.